uniflow/op/extract/load/html_op.py

```diff
@@ -0,0 +1,89 @@
+"""Extract HTML op."""
+import copy
+from typing import Sequence
+
+from uniflow.node import Node
+from uniflow.op.op import Op
+
+
+class ExtractHTMLOp(Op):
+    """Extract HTML Op Class."""
+
+    def __call__(self, nodes: Sequence[Node]) -> Sequence[Node]:
+        """Run Model Op.
+
+        Args:
+            nodes (Sequence[Node]): Nodes to run.
+
+        Returns:
+            Sequence[Node]: Nodes after running.
+        """
+        output_nodes = []
+        for node in nodes:
+            value_dict = copy.deepcopy(node.value_dict)
+            if "url" in value_dict:
+                import requests  # pylint: disable=import-outside-toplevel
```

Comment:
nit: Let's move `import requests` to the `__init__` of `ExtractHTMLOp` to avoid import for every `__call__`.